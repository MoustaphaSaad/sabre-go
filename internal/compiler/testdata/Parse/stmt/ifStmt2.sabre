if cond {
	a = b
} else {
	a = c
}