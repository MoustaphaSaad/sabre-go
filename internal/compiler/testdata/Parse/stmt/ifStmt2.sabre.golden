(IfStmt
  (IfStmt-Cond
    (IdentifierExpr IDENTIFIER(cond))
  )
  (IfStmt-Body
    (Block 1
      (AssignStmt
        (IdentifierExpr IDENTIFIER(a))
        =
        (IdentifierExpr IDENTIFIER(b))
      )
    )
  )
  (IfStmt-Else
    (Block 1
      (AssignStmt
        (IdentifierExpr IDENTIFIER(a))
        =
        (IdentifierExpr IDENTIFIER(c))
      )
    )
  )
)
